orangecontrib/text/import_documents.py

```diff
@@ -284,6 +287,8 @@ def _read_meta_data(self):
                 content = data.content
                 if isinstance(content, dict):
                     content = pd.DataFrame(content, index=[0])
+                # if reader is YamlMetaReader:
+                #     content = content.replace("None", np.nan)
```

Comment:
@VesnaT Should we keep this? Currently, null values in yamls are treated as "None" (strings).

Reply:
I'd rather put fixes like this inside the `YamlMetaReader`.
Besides, I'm not sure the fix works correctly for the`StringVariable`s.

Reply:
Something like this could work:
```
class YamlMetaReader(Reader):
    ext = [".yaml"]

    def read_file(self):
        with open(self.path, "r") as f:
            self.content = yaml.safe_load(f)
            for k in self.content:
                if self.content[k] is None:
                    self.content[k] = ""
            print(self.content)
```